boards/Kconfig

```diff
@@ -28,6 +28,15 @@ endchoice
 rsource "shields/*/Kconfig.shield"
 
 menu "Board Options"
+config QEMU_ICOUNT
```

Comment:
the commit which has this change should be much earlier and before you start using QEMU_ICOUNT, for bisectability.

Reply:
@nashif update